### PR TITLE
Update README.md to include BazelPods

### PR DESCRIPTION
**Pull request by sergeykhliustin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ the ruleset.
 - Ten Ten
 - Tinder
 - Tokopedia
+- [BazelPods](https://github.com/sergeykhliustin/BazelPods)
```
Comment (brentleyjones) [inline README.md:78]: Can you move this up to be alphabetical? Also, can you remove the link here, and add a permalink to https://github.com/sergeykhliustin/BazelPods/blob/main/xcodeproj/BUILD in the examples README?

Comment (sergeykhliustin) [reply README.md]: done

File: examples/README.md [hunk 2]
```diff
@@ -23,3 +23,4 @@ latest version on each respective repository.
 - [rules_apple iOS app tutorial](https://github.com/bazelbuild/rules_apple/blob/master/doc/tutorials/ios-app.md)
 - [SwiftLint](https://github.com/realm/SwiftLint/blob/325d0ee1e44a87fc82afeb874b83ceb82f6728cf/BUILD#L113-L142)
 - [SwiftUI iOS App with Bazel template](https://github.com/mattrobmattrob/bazel-ios-swiftui-template/blob/666640b796f347b62b8e5878e00c2d2f44c247cc/BUILD.bazel#L9-L18)
+- [BazelPods](https://github.com/sergeykhliustin/BazelPods/blob/main/xcodeproj/BUILD)
```
Comment (brentleyjones) [inline examples/README.md:26]: Can you move this up to just after `BazelCon 2022 Demo` (to keep it alphabetical)? Also, can you use `https://github.com/sergeykhliustin/BazelPods/blob/f77e705c46ae73453e8df65c0ad54f67a4ac3f96/xcodeproj/BUILD` as the link?

Comment (sergeykhliustin) [reply examples/README.md]: done

**Review by brentleyjones — APPROVED** [review]

**Pull request merged**